nodestream/pipeline/pipeline_file_loader.py

```diff
@@ -4,6 +4,8 @@
 
 from yaml import SafeLoader, load
 
+from nodestream.pipeline.scope_config import ScopeConfig
```

Comment:
this is a nitpick, but the rest of the imports are relative. Can we change this one to be?

Reply:
Sure, will do!